Mathlib/Analysis/Normed/Ring/Basic.lean

```diff
@@ -736,7 +730,44 @@ theorem norm_prod (s : Finset β) (f : β → α) : ‖∏ b ∈ s, f b‖ = ∏
 theorem nnnorm_prod (s : Finset β) (f : β → α) : ‖∏ b ∈ s, f b‖₊ = ∏ b ∈ s, ‖f b‖₊ :=
   map_prod nnnormHom.toMonoidHom f s
 
-end NormedCommRing
+end SeminormedCommRing
+
+section NormedAddCommGroup
+
+variable [NormedAddCommGroup α] [MulOneClass α] [NormMulClass α] [Nontrivial α]
```

Comment:
style nit:
```suggestion
section NormedAddCommGroup
variable [NormedAddCommGroup α] [MulOneClass α] [NormMulClass α] [Nontrivial α]
```

I think this blank line convention is a mathportism, and putting the variables next to the section (as we did in mathlib3) makes it easier to spot what the section is about; but I don't think we have official policy on this.